### DISCUSSION TITLE
yitaoli/posts/blog-hugo/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#47] [General]
# yitaoli/posts/blog-hugo/

为什么决定自建博客由于不想认证身份，也无法收到国内手机号验证码，微信公众号和豆瓣都上不去了。作为内容的创作者，却要看平台脸色这种事情也是受够

https://yitaoli2023.github.io/yitaoli/posts/blog-hugo/

**Comment by sommer0708** [comment]
乡友你好~同为github搭建blog的选手🙋我来踩踩！
PS：这个reaction功能好可爱！好想移植到我的blog里去！

**Reply by yitaoli2023** [reply]
谢谢乡友的支持！大家一起折腾起来哈哈😆

**Reply by zhenda-hub** [reply]
为了不看脸色， nice
